main.go

```diff
@@ -324,3 +325,12 @@ func main() {
 	log.Donef("Done")
 	log.Printf("Took: " + time.Since(startTime).String())
 }
+
+func isSameStack(archiveStackID string, currentStackID string) bool {
+	// TODO This check is a temporary solution to support GEN2 VMs having different ids for same stack types
+	if r, err := regexp.Compile("^(.+)-gen2.*$"); err == nil {
```

Comment:
You can use regexp.MustCompile

Reply:
I don't want it to panic if it can't parse the expression